python/ray/actor.py

```diff
@@ -357,7 +357,8 @@ def _remote(self,
                 is_direct_call=None,
                 max_concurrency=None,
                 name=None,
-                detached=False):
+                detached=False,
+                is_async=False):
```

Comment:
Can you add a unit test that covers asyncio? I think it has to be in one of the existing asyncio suites that only run in python 3.